modules/contact-form/grunion-contact-form.php

```diff
@@ -1252,6 +1253,7 @@ function process_submission() {
 
 		// Initialize all these "standard" fields to null
 		$comment_author_email = $comment_author_email_label = // v
+		$comment_author_telephone = $comment_author_telephone_label = // v
```

Comment:
These are only for standard comment fields, no?  Telephone isn't a standard comment field ...